library.gradle

```diff
@@ -7,6 +7,17 @@ android {
     compileSdkVersion compileVersion
     buildToolsVersion buildToolsVersion
 
+    flavorDimensions "extraAPIs"
+    productFlavors {
+        defaults {
+            dimension "extraAPIs"
+        }
+        integrationTest {
+            dimension "extraAPIs"
+            minSdkVersion 21 // Needed because mockk-android requires API 21
```

Comment:
Note how these are running on android 21. Our previous integration tests (that still exist) run in android 16. I think it's worth to keep tests running in Android 16 to ensure compatibility, but mockk-android doesn't have versions supporting Android < 21 afaik... We could just not use mockks and use fakes or something for these tests